src/main/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/indexstatemanagement/transport/action/removepolicy/RemovePolicyRequest.kt

```diff
@@ -0,0 +1,33 @@
+package com.amazon.opendistroforelasticsearch.indexmanagement.indexstatemanagement.transport.action.removepolicy
+
+import org.elasticsearch.action.ActionRequest
+import org.elasticsearch.action.ActionRequestValidationException
+import org.elasticsearch.common.io.stream.StreamInput
+import org.elasticsearch.common.io.stream.StreamOutput
+import java.io.IOException
+import java.util.Collections
+
+class RemovePolicyRequest : ActionRequest {
+
+    val indices: List<String>
+
+    constructor(
+        indices: List<String>
+    ) : super() {
+        this.indices = indices
+    }
+
+    @Throws(IOException::class)
+    constructor(sin: StreamInput) : this(
+        Collections.unmodifiableList(sin.readStringList())
+    )
+
+    override fun validate(): ActionRequestValidationException? {
+        return null
```

Comment:
Should we reject it if the indices list is empty?

Reply:
we handled the empty indices list [here](https://github.com/opendistro-for-elasticsearch/index-management/pull/293/files#diff-a8ed1e23c02d2a9a8a26dbe9ceaa433eR47)

Reply:
now handle in the request, our RemovePolicyRequest should take at least one index to work